_posts/2019-01-25-lambda-to-openfaas-with-skill.md

```diff
@@ -0,0 +1,280 @@
+---
+title: "A snowman's journey: Lambda to OpenFaaS"
+description: Richard Gee showcases a holiday project which demonstrates adding voice control to OpenFaaS functions using an Alexa skill and moving the handling function out of AWS Lambda onto OpenFaaS alternatives.
+date: 2019-01-25
+image: /images/lambda-to-openfaas-with-skill/background-brown-christmas.jpg
```

Comment:
BIN +2.18 MB images/lambda-to-openfaas-with-skill/background-brown-christmas.jpg

This image is too large, please resize 1600 on the width and export at ~= 250KB.